### ISSUE TITLE
Inference results in weird values

### ISSUE THREAD

**Issue by artyompal** [osmr/imgclsmob#19]
Hi,

I tried to use ImageNet classifier with a pretrained model SE-ResNext50. I added just a Softmax activation. It results in values with mean 0.01 and range about 0.005-0.02, which don't seem like proper softmax predictions. Actual results are incorrect.

If I change 1 line to use torchvision.models.resnet50, inference works like a charm: it detects dogs, cars, etc. Is there anything I'm missing? I'm using pytorchcv==0.0.45.

Btw, maybe you should add `__version__` into the package.

Cheers!

**Comment by artyompal** [comment]
Repro:
wrong https://github.com/artyompal/google_landmark_2019/blob/master/imagenet_classifier.py
right https://github.com/artyompal/google_landmark_2019/blob/master/imagenet_classifier2.py

**Comment by artyompal** [comment]
If I use pretrainedmodels.se_resnext50_32x4d instead, it also works. So I think there's a problem with models in this repo, specifically with the last FC layer. Probably no-one noticed it because nobody uses the last layer, it's usually all about fine-tuning.

**Comment by osmr** [comment]
What if you try ` model = get_model('seresnext50_32x4d', pretrained=True).cuda()`?

**Comment by osmr** [comment]
If the advice does not help, please reopen the issue.

**Comment by artyompal** [comment]
Oh thanks, that was actually a pretty stupid bug.

**Comment by osmr** [comment]
You're welcome:)
